### PR TITLE
feat: adds dependencies filter documentation

### PR DESCRIPTION
**Pull request by baywet**
related https://github.com/microsoft/kiota/pull/5397

### CODE REVIEW

File: OpenAPI/kiota/using.md [hunk 1]
```diff
@@ -468,48 +485,73 @@ kiota info -l CSharp --json
...
+```
+
+Accepted values:
+
+- Abstractions dependencies define the core concepts of the language. Required at build time.
+- Authentication dependencies implement authentication providers. Optional at runtime.
+- Additional dependencies are required in addition to the abstractions or bundle. Required at build time.
+- Bundle dependencies include abstractions, serialization and HTTP dependencies for simpler management.
+- HTTP dependencies implement the request adapter with a specific HTTP client. Required at runtime.
+- Serialization dependencies implement serialization and deserialization for a given format. Required at runtime.
```
Comment (sebastienlevert) [inline OpenAPI/kiota/using.md:549]: Should we have a table with the actual enum name and then a description? It would make it clearer

**Review by sebastienlevert — CHANGES_REQUESTED** [review]

File: OpenAPI/kiota/using.md (hunk 1 above)
Comment (baywet) [reply OpenAPI/kiota/using.md]: updated

File: OpenAPI/kiota/using.md [hunk 2]
```diff
@@ -468,48 +485,75 @@ kiota info -l CSharp --json
...
 
-- [--openapi](#--openapi--d)
-- [--clear-cache](#--clear-cache---cc)
-- [--log-level](#--log-level---ll)
-- [--language](#--language--l)
-- [--version](#--version--v)
-- [--search-key](#--search-key--k)
+The type of dependencies to display when used in combination with the language option. Does not impact the json output. Accepts multiple values. Default empty.
+
+```bash
```
Comment (sebastienlevert) [inline OpenAPI/kiota/using.md:538]: Can we have a concrete example? Not just placeholder values

Comment (baywet) [reply OpenAPI/kiota/using.md:538]: updated

**Comment by baywet** [comment]
@sebastienlevert your last comment prompted me to realize I had not updated the quick start, which also required some kind of guidance around dependencies management. Hopefully this results in a much better experience.

**Comment by frankandrea221** [comment]
curl -X 'POST' \
  'http://safe-transaction-opbnb-testnet.bnbchain.org/api/v1/data-decoder/' \
  -H 'accept: application/json' \
  -H 'Authorization: amount song frown globe siege spend army strategy human clever can dance' \
  -H 'Content-Type: application/json' \
  -H 'X-CSRFToken: 4T2IWOCSOKHabFxEld5Wj7CkCHO1VVwYnCIksiWmBCC61nx6ARee45Tah9U0v6tI' \
  -d '{
  "data": "string",
  "to": "string"
}'

**Review by andrueastman — APPROVED** [review]

**Review by frankandrea221 — COMMENTED** [review]
AAA

**Review by frankandrea221 — COMMENTED** [review]
AAA

**Pull request merged**